kotlin/browse-ogc-api-feature-service/src/main/java/com/esri/arcgisruntime/sample/browseogcapifeatureservice/MainActivity.kt

```diff
@@ -0,0 +1,246 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.esri.arcgisruntime.sample.browseogcapifeatureservice
+
+import android.graphics.Color
+import android.os.Bundle
+import android.view.MotionEvent
+import android.view.View
+import android.widget.ArrayAdapter
+import android.widget.EditText
+import android.widget.ListView
+import androidx.appcompat.app.AppCompatActivity
+import androidx.coordinatorlayout.widget.CoordinatorLayout
+import com.esri.arcgisruntime.ArcGISRuntimeEnvironment
+import com.esri.arcgisruntime.data.FeatureTable
+import com.esri.arcgisruntime.data.OgcFeatureCollectionTable
+import com.esri.arcgisruntime.data.QueryParameters
+import com.esri.arcgisruntime.data.ServiceFeatureTable
+import com.esri.arcgisruntime.geometry.GeometryType
+import com.esri.arcgisruntime.layers.FeatureLayer
+import com.esri.arcgisruntime.layers.OgcFeatureCollectionInfo
+import com.esri.arcgisruntime.layers.OgcFeatureService
+import com.esri.arcgisruntime.mapping.ArcGISMap
+import com.esri.arcgisruntime.mapping.BasemapStyle
+import com.esri.arcgisruntime.mapping.view.DefaultMapViewOnTouchListener
+import com.esri.arcgisruntime.mapping.view.MapView
+import com.esri.arcgisruntime.sample.browseogcapifeatureservice.databinding.ActivityMainBinding
+import com.esri.arcgisruntime.symbology.Renderer
+import com.esri.arcgisruntime.symbology.SimpleFillSymbol
+import com.esri.arcgisruntime.symbology.SimpleLineSymbol
+import com.esri.arcgisruntime.symbology.SimpleMarkerSymbol
+import com.esri.arcgisruntime.symbology.SimpleRenderer
+import com.google.android.material.floatingactionbutton.FloatingActionButton
+
+// URL to the OAFeat service.
+private const val serviceUrl = "https://demo.ldproxy.net/daraa"
+
+class MainActivity : AppCompatActivity() {
+
+    private val activityMainBinding by lazy {
+        ActivityMainBinding.inflate(layoutInflater)
+    }
+
+    private val mapView: MapView by lazy {
+        activityMainBinding.mapView
+    }
+
+    private val serviceEditText: EditText by lazy {
+        activityMainBinding.serviceEditText
+    }
+
+    private val layerFAB: FloatingActionButton by lazy {
+        activityMainBinding.layerFAB
+    }
+
+    private val featureCollectionTitleListView: ListView by lazy {
+        activityMainBinding.layerList
+    }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(activityMainBinding.root)
+
+        // authentication with an API key or named user is required to access basemaps and other
+        // location services
+        ArcGISRuntimeEnvironment.setApiKey(BuildConfig.API_KEY)
+
+        // initialize the text box with a service url
+        serviceEditText.setText(serviceUrl)
+        // create a map with topographic basemap
+        mapView.map = ArcGISMap(BasemapStyle.ARCGIS_TOPOGRAPHIC)
+
+        // load the service
```

Comment:
it's not initially clear what service this comment refers to, is it the above mentioned location services or the OGC feature service: suggest clarification below:
```suggestion
        // load the OGC feature service
```